frontend/src/metabase/nav/containers/SearchBar.jsx

```diff
@@ -0,0 +1,117 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import { Flex } from "grid-styled";
+import styled from "styled-components";
+import { space } from "styled-system";
+import { t } from "ttag";
+
+import { color, lighten } from "metabase/lib/colors";
+
+import Icon from "metabase/components/Icon";
+import OnClickOutsideWrapper from "metabase/components/OnClickOutsideWrapper";
+
+import { DefaultSearchColor } from "metabase/nav/constants";
+
+const ActiveSearchColor = lighten(color("nav"), 0.1);
+
+const SearchWrapper = Flex.extend`
+  background-color: ${props =>
+    props.active ? ActiveSearchColor : DefaultSearchColor};
+  border-radius: 6px;
+  flex: 1 1 auto;
+  max-width: 50em;
+  align-items: center;
+  color: white;
+  transition: background 300ms ease-in;
+  &:hover {
+    background-color: ${ActiveSearchColor};
+  }
+`;
+
+const SearchInput = styled.input`
+  ${space} background-color: transparent;
+  width: 100%;
+  border: none;
+  color: white;
+  font-size: 1em;
+  font-weight: 700;
+  &:focus {
+    outline: none;
+  }
+  &::placeholder {
+    color: ${color("text-white")};
+  }
+`;
+
+const SEARCH_FOCUS_ELEMENT_WHITELIST = new Set(["BODY", "A"]);
```

Comment:
Maybe change to `SEARCH_FOCUS_ELEMENT_ALLOWLIST` while we're updating the file.

Reply:
Ah good catch! Definitely.